Dockerfile

```
@@ -45,6 +45,9 @@ ENV GITHUB_TOKEN ${GITHUB_TOKEN}
 ARG SENTRY_AUTH_TOKEN
 ENV SENTRY_AUTH_TOKEN ${SENTRY_AUTH_TOKEN}
 
+ARG SENTRY_RELEASE
```

will this be needed on coolify as well? will the build go okay without it or will it crash? default value?

---

Do we have the other sentry keys in coolify as well? Do we want those test envs in Sentry at all?

---

It should default to some random hash.